run_flake8.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -euo pipefail
+
+directory=${1:-.}
+flake8 --exclude thirdparty,build --ignore E501,W504 --per-file-ignores="steps.py:F811" "${directory}"
```

Comment:
I'm happy with ignoring [E501](https://www.flake8rules.com/rules/E501.html), but asking for an explanation of ignoring [W504](https://www.flake8rules.com/rules/W504.html) and [F811](https://www.flake8rules.com/rules/F811.html).

Reply:
Flake8 has both [W503](https://www.flake8rules.com/rules/W503.html) and [W504](https://www.flake8rules.com/rules/W504.html) enabled by default and they contradict each other. We have to choose one of them to use, in this case I chose to use W503 and ignore W504 to have the binary operators at the end of the line.

[F811](https://www.flake8rules.com/rules/F811.html) is only ignored in steps.py as it defines the `behave` framework steps. This file has the same `step_impl` function name for each step and `flake8` detects it as a redefinition.

Reply:
Between W503 and W504, I would go for the other one based on this: https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator
> In Python code, it is permissible to break before or after a binary operator, as long as the convention is consistent locally. For new code Knuth's style is suggested.

Knuth's style is break before the operator.

Reply:
Yes, in hindsight it makes more sense to go with that, even the [flake8 rule](https://www.flake8rules.com/rules/W503.html) mentions that this goes against the pep8 recommendation. I added the change in f417b932e4f76b74351c78739eefd1d462dac4b4